lib/client/query_service.dart

```diff
@@ -37,8 +37,14 @@ class QueryService extends DefaultService {
   ///
   /// Use this with care, all response is stored in memory.
   /// Result CSV format can be changed using [dialect].
-  Future<String> queryRaw(String fluxQuery, {Dialect? dialect}) async {
-    var query = Query(dialect: dialect, query: fluxQuery);
+  Future<String> queryRaw(fluxQuery,
+      {Dialect? dialect, Map<String, Object>? params}) async {
+    var query = fluxQuery is Query ? fluxQuery : Query(query: fluxQuery);
+
+    query.params = params ?? query.params;
+    query.dialect = query.dialect ?? DEFAULT_dialect;
+    query.dialect = dialect ?? query.dialect;
```

Comment:
This probably breaks the test https://app.circleci.com/pipelines/github/influxdata/influxdb-client-dart/178/workflows/656f2e50-d639-4090-a5f9-96d36d8d2e15/jobs/857/tests#failed-test-0

Reply:
Dialect can be null in `queryRaw`. In that case raw response is without annotations

like: 
```
',result,table,_time,_value,_field,_measurement,location\r\n'
',_result,0,1970-01-01T00:00:00.000000001Z,999,value,temperature,north\r\n'
',_result,1,1970-01-01T00:00:00.000000002Z,111,value,temperature,south\r\n'
'\r\n'
```